build/i2c/src/lib.rs

```diff
@@ -1254,6 +1268,36 @@ impl ConfigGenerator {
 
                 let out = self.generate_device(device, 16);
                 writeln!(&mut self.output, "({}, {})\n        }}", out, index)?;
+
+                if which == PowerDevices::PMBus {
+                    let phases = if let Some(power) = &device.power {
+                        if let Some(phases) = &power.phases {
+                            if *index >= phases.len() {
```

Comment:
I think this should be unreachable, because we check that `power.rails` and `power.phases` are the same size above.  (Coupling the rails and phases in the TOML files would also make this kind of error impossible!)

Reply:
Yes, thank you.